lib/routes/main_page.dart

```diff
@@ -319,6 +319,8 @@ class _MainPageState extends State<MainPage> with TickerProviderStateMixin {
     }
   }
   Future<void> sync() async {
-    await locator<SyncRoutine>().syncNotes();
+    Future(() {
+      locator<SyncRoutine>().syncNotes();
```

Comment:
What the hell is this, why did you do this, the one before was the same but much cleaner